package-scripts.js

```diff
@@ -46,7 +46,7 @@ module.exports = {
     },
     // default is run when you run `nps` or `npm start`
     default: 'next start',
-    dev: 'next',
+    dev: crossEnv('NODE_ENV=development next'),
```

Comment:
With this we'll be able to use glamor's `label` functionality :tada:

glamorous will do this automatically in the next version